src/main/java/net/revelc/code/impsort/ImpSort.java

```diff
@@ -35,6 +35,9 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.plugin.logging.Log;
```

Comment:
For reusability of this code, it was the original intent to avoid using the maven-specific types in the `net.revelc.code.impsort` package, and only use the maven-specific imports in the `net.revelc.code.impsort.maven.plugin` sub-package. This is why we throw `ImpSortException` in this class, and then convert it to a Maven exception type in the `AbstractImpSortMojo` class.

Reply:
Aha, I see.
That's why old code use System.out not logger to print problems.

Reply:
You're right, and I see now that we're already using System.out in the other location just below this code, so I think we can just stick to that, which you originally did. Sorry for the churn, and the delay in getting this wrapped up.